pages/linux/nl.md

```diff
@@ -13,3 +13,11 @@
 - Number only the body lines that match a basic regular expression (BRE) pattern:
 
 `nl -b p'FooBar[0-9]' {{file}}`
+
+- Number all lines including blank lines:
```

Comment:
So example #1 on this page doesn't number blank lines?

Reply:
No example 1 doesn't number blank lines.

Reply:
Perhaps we could expand the description of the first example to reflect this then?

Reply:
done.